### ISSUE TITLE
✅ 🐞 Popup links throw a "Invalid Selector" warning in the browser console

### ISSUE THREAD

**Issue by mcrowell** [elementor/elementor#29137]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Elementor 3.25.3 and Pro 3.25 is giving the console error when using a Popup:
frontend.min.js?ver=3.25.3:2 Invalid selector: '#elementor-action%3Aaction%3Dpopup%3Aclose%26settings%3DeyJkb19ub3Rfc2hvd19hZ2FpbiI6IiJ9'

![Popup-Error](https://github.com/user-attachments/assets/dfcc3d76-d599-44a1-ac51-acd24ff36b81)

### Steps to reproduce

1. Create a Popup Template and assign to page.
2. View browser console error.

### Expected behavior

I don't remember seeing this error before 3.25 in the console when using a Popup.

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MySQL Community Server - GPL v8.0.40
	PHP Version: 8.1.30
	PHP Memory Limit: 768M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 300M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.2
	Site URL: https://wickydemo.com/wickydesign
	Home URL: https://wickydemo.com/wickydesign
	WP Multisite: No
	Max Upload Size: 300 MB
	Memory limit: 40M
	Max Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: America/New_York	
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 2.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.1.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.25.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.25.0
		Author: Elementor.com

	Wordfence Security
		Version: 7.11.7
		Author: Wordfence


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Active by default
	Additional Custom Breakpoints: Inactive
	Container: Active by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active
	Optimized Markup: Active
	Conditionally load Swiper CSS files: Inactive by default
	Plugin Onboarding: Active by default
	CSS Smooth Scroll: Active by default
	Default to New Theme Builder: Inactive
	Header & Footer: Inactive
	Elementor Home Screen: Active by default
	Landing Pages: Inactive
	Nested Elements: Active
	Editor Top Bar: Active by default
	Pages Panel: Inactive by default
	Element Caching: Inactive
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Launchpad Checklist: Inactive by default
	Menu: Active

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by toineenzo** [comment]
Have seen this problem as well but doesn't only happen for me when creating a Popup

**Comment by reedock** [comment]
Same here

**Comment by kweidner14** [comment]
Also have the same issue. Popup is displaying incorrectly as well. Had to revert to previous version of Elementor and Pro. After revert, warnings have disappeared and popup is displaying properly.

**Comment by jdpoccorie** [comment]
Perhaps this solution will help you figure out the problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by toineenzo** [comment]
A fix will be released soon I think:
https://github.com/elementor/elementor/pull/29155
https://github.com/elementor/elementor/pull/29156

**Comment by meacodes** [comment]
same here frontend.min.js?ver=3.25.3:2 Invalid selector: '#0-toc-title' in conflict with betterdoc plugin

**Comment by diegomontejano** [comment]
Same here

**Comment by babsg1983** [comment]
Same issue. Any status on resolving this?

**Comment by nicholaszein** [comment]
Hi @mcrowell, and others!

Thank you for submitting your issue! 🙏 

We are aware of this issue and our team is already working on a fix. 

Best regards

**Comment by nicholaszein** [comment]
Hi everyone!

We have great news! 🙌 

### 📢 We're happy to announce that this issue was solved with the release of Elementor Core v3.25.4, and Elementor Pro v3.25.2! 🥳 

✅ Feel free to check it out and update both your plugins to the newest versions!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
